archive-press_quote.php

```diff
@@ -0,0 +1,71 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+if( have_posts() ) {
+  $i = 0;
+
+  while( have_posts() ) {
+    the_post();
+    $quote = get_post_meta($post->ID, '_igv_quote_text', true);
+    $link = get_post_meta($post->ID, '_igv_quote_link', true);
+
+    $title = get_the_title();
```

Comment:
They would still have to make the title field a unique identifier of some sort or it would just be lots of untitled posts. I think showing the quote in the admin is a better solution and similar amount of work from rn I think.